libmobilecoin/include/chacha20_rng.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+#ifndef MC_CHACHA20_RNG_H_
+#define MC_CHACHA20_RNG_H_
+
+#include "common.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct _ChaCha20Rng ChaCha20Rng;
+
+ChaCha20Rng* MC_NULLABLE mc_chacha20_rng_create_with_long(uint64_t value);
```

Comment:
missing function documentation comments, also formatting looks a lil different from other header files should be adjusted to match style in other files.

Reply:
Added comments in a subsequent commit, and formatting was updated in a subsequent commit as well.
Adding the /* ======= NAME ========= */ comment line under #include "common.h" (last change not committed yet b/c want to avoid having to cherry-pick too much... so going to bundle the review updates together).